src/tapis-ui/utils/range.ts

```diff
@@ -0,0 +1,7 @@
+export const range = (size: number, startAt: number = 0): Array<number> => {
```

Comment:
I think this can be more efficient:

```
[...Array(5).keys()].splice(0, startAt)
```

If this is going to be a utility function, it's going to need a unit test. Alternatively, just install this npm package which also does range with step: 

https://www.npmjs.com/package/range

Reply:
Removed range util